packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## v3.1.0-dev (Aug 08, 2024)
-
 - Add `defaultDnt` to support setting the dnt flag for SLAS. Upgrade `commerce-sdk-isomorphic` to v3.1.1 [#1979](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1979)
+- Added useDNT hook to commerce-sdk-react [#1989](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1989)
 - Read the DNT preference from the `dw_dnt` cookie set by the `useDnt` hook. If the cookie is not set, `defaultDnt` will be used to set the DNT flag. [#1987] (https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1987)
```

Comment:
Since the CI enforces it, we had to add it in, and Alex said to put this under the 3.1.0-dev version. But on the final merge to `develop` we can evaluate if we need to combine these PRs into one.